std_document_store_dev/js/docstore_workflow_debug.js

```diff
@@ -0,0 +1,136 @@
+/* Haplo Platform                                     http://haplo.org
+ * (c) Haplo Services Ltd 2006 - 2016    http://www.haplo-services.com
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.         */
+
+P.onLoad = function() {
+    if(!O.PLUGIN_DEBUGGING_ENABLED) {
+        throw new Error("DEV PLUGIN std_document_store_dev LOADED ON NON-DEV SERVER");
+    }
+};
+
+if(O.PLUGIN_DEBUGGING_ENABLED) {
+
+    // UI for showing/hiding implemented by std_workflow_dev
+    var showDebugTools = function() {
+        return (O.PLUGIN_DEBUGGING_ENABLED &&
+            O.currentAuthenticatedUser &&
+            O.currentAuthenticatedUser.isSuperUser &&
+            O.currentAuthenticatedUser.id === 3 && // SUPPORT only for now
```

Comment:
Perhaps we should just let anyone use this? Some developers aren't going to be logging in via a dev portal which gets them SUPPORT.

Reply:
Agreed. My intent here was being overly cautious until it's been used a bit and then dropping such a restriction.